### ISSUE TITLE
Wine cellar HWS77GDAU1 double zone temperature setting

### ISSUE THREAD

**Issue by websy06** [Andre0512/hon#238]
**Describe the bug**

Hi,
Thanks a lot for your great job with Haier.  May I report the following bug ?
I have a HWS77GDAU1 wine cellar with a double zone.
When you change the temperature/program (sparkling, red wine, whitewine, ...) on one of the zones, the other parameters reset to the default parameter on the other zone.

**Expected behavior**
When you change the temperature/programe of one zone, the other one should not change and should keep the values already set.

Home Assistant
    Core 2024.8.2
    Supervisor 2024.08.0
    Operating System 13.1
    Frontend 20240809.0 ⸱ legacy
-hOn Integration Version: 0.14.0

**Comment by tomazzive** [comment]
I have the same issue. I tried to find a logic, but couldnt find any. I do notice that the temp setting of one zone is duplicated to the other zone. It can also be the setting of a program. Its both Z1 that is set to the temp of Z2 and the way around. I had this issue for two weeks. 5 times now. Not good for my wine

**Comment by tomazzive** [comment]
I think I found the logic: when i turn off the light with the hacs integration, bot zones were set to 5° (= bubbels program) 
In fact I added a automation that turns the light on at sunset and turns it off at 22h. So thats why I had the issue since 2 weeks

**Comment by ramundomario** [comment]
Discovered this weird issue today too with an HWS42GDAU1
Unfortunately the upper zone got the same temperature that the lower one 😞 

In addition the current temperature reported are wrong, equal to `target_temperature`

**Comment by Coldrush** [comment]
Hi there! I'm currently considering buying the HWS42GDAU1 and I'm curious if this bug has been resolved?

**Comment by Radjone** [comment]
Hey guys

Same issue here. When i turn on and off the light, 5degrees

Have you found a solution ?

**Comment by tomazzive** [comment]
No. I don't use the integration to change settings or turn on/off the lights. Only for monitoring. Pitty. 
Or were your suggestions changed in the code, @ramundomario ?
